django/db/migrations/state.py

```diff
@@ -680,10 +680,14 @@ def clone(self):
         """Return a clone of this registry."""
         clone = StateApps([], {})
         clone.all_models = copy.deepcopy(self.all_models)
-        clone.app_configs = copy.deepcopy(self.app_configs)
-        # Set the pointer to the correct app registry.
-        for app_config in clone.app_configs.values():
+
+        clone.app_configs = {}
```

Comment:
`app_configs` is initialized to an empty dict, so this line should be unnecessary :thinking: